safe/gui/tools/options_dialog.py

```diff
@@ -412,7 +418,7 @@ def on_toolNorthArrowPath_clicked(self):
 
     # noinspection PyPep8Naming
     @pyqtSignature('')  # prevents actions being handled twice
```

Comment:
Can'we remove these decorator too? I guess if we use explicit connect, it's fine, no? It's only a question.

Reply:
I think we can. I never use this decorator actually.